File: QTD.SPA/src/app/_DtoModels/Procedure/Procedure_review/submitProcedureReviewDto.ts
export class SubmitProcedureReviewDto{
    procedureReviewId:string;
    response: string|null;
    comments: string;

    constructor(id: string, res: string | null, comment: string){
        this.procedureReviewId = id;
        this.response = res;
        this.comments = comment;
    }
}

File: QTD2.Infrastructure/Model/ProcedureReview/SubmitProcedureReviewDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.ProcedureReview
{
    public class SubmitProcedureReviewDto
    {
        public int ProcedureReviewId { get; set; }
        public string? Response { get; set; }
        public string Comments { get; set; }
    }
}
